### PR TITLE
Add pagination to homepage transactions

### LINKED ISSUE #60: Enable pagination to navigate to older transactions
Screen Shot 2022-05-10 at 5 54 08 PM

### PR DESCRIPTION
**Pull request by LuizAsFight**
Enable user to check older transactions with simple pagination

Closes #60

image
image

### CODE REVIEW

**Comment by LuizAsFight** [comment]
https://user-images.githubusercontent.com/8636507/167968989-bad28b25-9bb0-4e6e-b506-d13ddc8c5c57.mov

https://user-images.githubusercontent.com/8636507/167969017-4107415f-073b-42c6-b4a2-dd5a45ef8ce1.mov

File: src/pages/HomePage/RecentTransactions.tsx [hunk 1]
```diff
@@ -27,15 +36,42 @@ type Props = {
 type Ouputs = HomePageTransaction['outputs'];
 
 export function RecentTransactions({ transactions }: Props) {
```
Comment (luizstacio) [inline src/pages/HomePage/RecentTransactions.tsx:38]: Would be nice to use pagination directly on the GraphQL queries as our core has this capability.

If you have the network running locally you can access `http://localhost:4000/playground` to see the GraphQL Playground.

## Examples

### First query without cursor query the latest 5 transactions
```
query {
  transactions(last: 5) {
    pageInfo {
      hasPreviousPage
      hasNextPage
      startCursor
    }
    edges {
      node {
        id
      }
    }
  }
}
```

### Next page query passing the `startCursor`
```
query {
  transactions(last: 5, before: "5#0x8844e7be9a02d03a5681a5b280dc0fc9ac423f50d9c3c9cbfde370d7c11df3e8") {
    pageInfo {
      hasPreviousPage
      hasNextPage
      startCursor
    }
    edges {
      node {
        id
      }
    }
  }
}
```

**Review by luizstacio — CHANGES_REQUESTED** [review]

**Comment by luizstacio** [comment]
We have a issue with paginations when arriving to the last it will be fixed by: https://github.com/FuelLabs/fuel-core/pull/323/files

**Review by luizstacio — APPROVED** [review]
LGTM

**Pull request merged**